package.json

```
@@ -47,5 +47,10 @@
     "typescript": "^4.0.5"
   },
   "author": "wwwy3y3",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/keycloak/keycloak-nodejs-admin-client.git"
```

I believe we do not have to add the `git+` prefix here, so it can be omitted.
```suggestion
    "url": "https://github.com/keycloak/keycloak-nodejs-admin-client.git"
```

---

I used what I saw in one of the other keyclaok repos, but happy to remove.